tests/conftest.py

```diff
@@ -49,3 +49,13 @@ def s3_store(s3: str):
         aws_skip_signature=True,
         client_options={"allow_http": True},
     )
+
+
+@pytest.fixture
+def s3_store_config(s3: str):
```

Comment:
```suggestion
def s3_store_config(s3: str) -> S3ConfigInput:
```

Reply:
Correct, `allow_http` is a client option because it gets passed to the HTTP client, and is shared among all store implementations.